src/scripts/Selector/SelectorTable.js

```diff
@@ -65,22 +65,20 @@ export default class SelectorTable extends Selector {
 		result = Array.from({
 			length: dataCells.length / Object.keys(columnIndices).length,
 		}).map(_ => Object());
+		const offset = dataCells[0].cellIndex; // cellIndex = порядковому номеру элемента среди его одноуровневых элементов
+		const revColInd = Object.entries(columnIndices).map(([k, v]) => [v, k]); // Приводим к entries, а затем инвертируем объект columnIndices
+		const ind2names = Object.fromEntries(revColInd); // Получаем из инвертированного revColInd итерируемый список
 		dataCells.each((rowNum, dataCell) => {
-			if (dataCell.cellIndex === 0) {
-				console.log(
-					"%c Vertical rows can't have first column as data cell ",
-					'background: red; color: white;'
-				);
-			} else {
-				const headerName = SelectorTable.trimHeader(
-					$(dataCell).closest('tr').find('th, td')[0].innerText
-				);
+			const indRow = $(dataCell).closest('tr')[0].rowIndex; // Получаем индекс строки клетки
```

Comment:
Now this is an example of a good comment, it explains what is happening rather than repeating what is written in code.
But what is even better, since there is one more place where you use this idiom, we may refactor it into a function and give it some meaningful name
```javascript
	getCellRowIndex(tableCell) {
		return $(tableCell).closest('tr')[0].rowIndex;
	}
```